cluster/gce/gci/configure-helper.sh

```diff
@@ -1562,6 +1650,51 @@ function start-etcd-servers {
   prepare-etcd-manifest "-events" "4002" "2381" "100m" "etcd-events.manifest"
 }
 
+# Replaces the variables in the network-proxy manifest file with the real values, and then
+# copy the file to the manifest dir
+# $1: value for variable "cpulimit"
```

Comment:
I prefer the pattern `local cpulimt=$1` as the first few lines in the function, this documentation becomes unnecessary and the body of the function will become readable...